mssql-ballerina/tests/connection-ssl-test.bal

```diff
@@ -14,20 +14,57 @@
 // under the License.
 
 import ballerina/test;
+import ballerina/file;
 
 string sslDb = "SSL_CONNECT_DB";
+string trustStorePath = checkpanic file:getAbsolutePath("./tests/resources/keystore/client/client-truststore.p12");
```

Comment:
This is fine, but we can use the relative file path for test cases.

Reply:
Fixed in 67561e9